ui/app/src/views/config/ConfigView.tsx

```diff
@@ -11,22 +11,69 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Stack } from '@mui/material';
+import { ReactElement, useState } from 'react';
+import { Stack, Box } from '@mui/material';
 import Cog from 'mdi-material-ui/Cog';
+import Puzzle from 'mdi-material-ui/Puzzle';
 import { JSONEditor } from '@perses-dev/components';
-import { ReactElement } from 'react';
+import { MenuTab, MenuTabs } from '../../components/tabs';
+
 import AppBreadcrumbs from '../../components/breadcrumbs/AppBreadcrumbs';
 import { useConfigContext } from '../../context/Config';
 import { useIsMobileSize } from '../../utils/browser-size';
+import { PluginsList } from './PluginsList';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps): ReactElement | null {
+  const { children, value, index, ...other } = props;
+  return (
+    <div role="tabpanel" hidden={value !== index} id={`tabpanel-${index}`} aria-labelledby={`tab-${index}`} {...other}>
+      {value === index && <Box sx={{ pt: 2 }}>{children}</Box>}
+    </div>
+  );
+}
 
 function ConfigView(): ReactElement {
   const { config } = useConfigContext();
   const isMobileSize = useIsMobileSize();
+  const [tabIndex, setTabIndex] = useState(0);
+
+  const handleTabChange = (_event: React.SyntheticEvent, newValue: number): void => {
+    setTabIndex(newValue);
+  };
 
   return (
     <Stack sx={{ width: '100%', overflowX: 'hidden' }} m={isMobileSize ? 1 : 2} mt={1.5} gap={2}>
       <AppBreadcrumbs rootPageName="Configuration" icon={<Cog fontSize="large" />} />
-      <JSONEditor value={config} readOnly />
+
```

Comment:
Missing wrapper for bottom border + fix margin top:
```suggestion
   <Stack>
     <Stack
        direction="row"
        alignItems="center"
        justifyContent="space-between"
        sx={{ borderBottom: 1, borderColor: 'divider' }}
      >
```

Reply:
Thank you. fixed in 7e675b38273fdfec055b00c5d91e703387b2c75c